common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -162,6 +162,8 @@ class CapaFields(object):
                        scope=Scope.user_state, default={})
     input_state = Dict(help=_("Dictionary for maintaining the state of inputtypes"), scope=Scope.user_state)
     student_answers = Dict(help=_("Dictionary with the current student responses"), scope=Scope.user_state)
+    saved_answers = Boolean(help=_("Whether or not the answers have been saved since last submit"),
```

Comment:
Where is the saved answer actually stored? Why is it necessary to have an additional boolean instead of looking to see if a saved answer exists?